.github/ISSUE_TEMPLATE/release.md

```diff
@@ -89,7 +89,7 @@ git push upstream v0.6.0
 28.  - [ ] Aggregate all the changelog notices from the PRs and add them to the changelog file
 29.  - [ ] Bump the versions on master from `0.9.0-SNAPSHOT` to `0.10.0-SNAPSHOT`, `0.6.0.dev0` to `0.7.0.dev0`
 30.  - [ ] Apply changes to the minor_release and patch_release issue templates if they need any
-31.  - [ ] Open a PR on master with the changes in steps 29, 30, and 31. See [example PR here](https://github.com/bitcoindevkit/bdk-ffi/pull/317). Get a review and merge the PR.
+31.  - [ ] Open a PR on master with the changes in steps 28, 29, 30, and 31. See [example PR here](https://github.com/bitcoindevkit/bdk-ffi/pull/317). Get a review and merge the PR.
```

Comment:
leaving 31 here is sort of self-referencing. I suggest to only keep 28, 29, 30.

Reply:
removed 31, then just re-numbered the steps that followed-

Reply:
Oh I see. Yep I re-read what I wrote and I see how it sounded now. Yeah that works too!

Reply:
I think it's fairly clear those changes need to be merged. As long as it's clear to you as well that they need to be merged on master then it's cool.

Reply:
Yeah I think its clear to me now but I'll keep it in mind, but yeah I read it one way at first and updated the PR and re-read it now and it makes sense in a different way. Will merge this PR, but will keep in mind if it doesnt make sense next time to add it back or re-word it